docs/resources/connection.md

```diff
@@ -146,7 +158,9 @@ The following arguments are supported:
 
 - `handle` - (Required) A friendly identifier for your connection, and must be unique across your connections.
 - `plugin` - (Required) The name of the plugin.
-- `config` - (Optional) Configuration for the connection.
+- `config` - (Optional) JSON Configuration for the connection, stored in state (cannot be used with `config_wo`). Note: As secrets are not returned from the API, this may show perpetual config drift is secrets are included in this argument.
```

Comment:
```suggestion
- `config` - (Optional) JSON configuration for the connection, stored in state (cannot be used with `config_wo`). Note: As secrets are not returned from the API, this may show perpetual config drift is secrets are included in this argument.
```